hugo/content/research/archives/2017.md

```diff
@@ -1,14 +1,20 @@
 ---
-title: "DORA Reseach: 2017"
+title: "DORA Reseach: 2017 and earlier"
 date: 2017-10-01
 draft: false
-research_year: 2017
+research_year: 2017 and earlier
 sections_to_display:
 type: research_archives
 ---
 
-# DORA's Research Program: 2017
-DORA's research program continued in 2017. [Read the 2017 report here](/publications/pdf/state-of-devops-2017.pdf)
+# 2017 and earlier
+Prior to 2018, research was conducted in conjunction with Puppet, as an extension of their original State of DevOps research. Read the [history of Puppet's DevOps Reports](https://www.puppet.com/resources/history-of-devops-reports)
+
+#### State of DevOps Reports published in partnership with Puppet:
+- [2017 State of DevOps Report](pdf/state-of-devops-2017.pdf)
+- [2016 State of DevOps Report](pdf/state-of-devops-2016.pdf)
+- [2015 State of DevOps Report](pdf/state-of-devops-2015.pdf)
+- [2014 State of DevOps Report](pdf/state-of-devops-2014.pdf)
 
```

Comment:
these all 404 I think they should link to /publications/pdf/state-of-devops-{{YYYY}}.pdf